docs/source/get_started/installation.rst

```diff
@@ -29,6 +29,42 @@ We recommend installing RDKit with deepchem if you use conda.
 
     conda install -y -c conda-forge rdkit
 
+Latest version
```

Comment:
We should merge this with the nightly build docs as indicated in the other comments above